Lib/fontmake/font_project.py

```diff
@@ -270,18 +263,20 @@ def run_from_glyphs(
             glyphs_source = self.preprocess(glyphs_file)
             glyphs_file = UnicodeIO(glyphs_source)
 
-        print('>> Building UFOs from Glyphs source')
-        ufos = self.build_ufos(
-            glyphs_file, interpolate, masters_as_instances, family_name)
-        self.run_from_ufos(
-            ufos, is_instance=(interpolate or masters_as_instances), **kwargs)
+        print('>> Building designspace from Glyphs source')
+        _, designspace_path, instance_data = self.build_master_ufos(
```

Comment:
The FontProject build_master_ufos method requests instances, so this should be working now.